src/analysis/retail/shaman/restoration/modules/dragonflight/Tier30.tsx

```diff
@@ -0,0 +1,240 @@
+import ITEMS from 'common/ITEMS';
+import SPELLS from 'common/SPELLS';
+import talents from 'common/TALENTS/shaman';
+import { TIERS } from 'game/TIERS';
+import Analyzer, { Options, SELECTED_PLAYER, SELECTED_PLAYER_PET } from 'parser/core/Analyzer';
+import { calculateEffectiveHealing } from 'parser/core/EventCalculateLib';
+import Events, { CastEvent, HealEvent } from 'parser/core/Events';
+import {
+  getSwellingRainHealingWaves,
+  getTidewatersHealingEvents,
+} from '../../normalizers/Tier30Normalizer';
+import ChainHealNormalizer from '../../normalizers/ChainHealNormalizer';
+import { SHAMAN_T30_ID } from 'common/ITEMS/dragonflight';
+import { SpellLink } from 'interface';
+import ItemSetLink from 'interface/ItemSetLink';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+import STATISTIC_ORDER from 'parser/ui/STATISTIC_ORDER';
+import Statistic from 'parser/ui/Statistic';
+import { formatNumber, formatPercentage } from 'common/format';
+import ItemHealingDone from 'parser/ui/ItemHealingDone';
+import { CAST_BUFFER_MS } from '../../constants';
+
+const TIER_4PC_RAINSTORM_PER_STACK = 0.01;
+const TIER_4PC_SWELLING_RAIN_CHAINHEAL_PER_STACK = 0.02;
+const TIER_4PC_SWELLING_RAIN_HSHW_PER_STACK = 0.1;
```

Comment:
Minor but probably don't need the `TIER_4PC_` prefix since its on all of the constants.